doc/man/bitcoin-cli.1

```diff
@@ -69,16 +71,16 @@ Timeout during HTTP requests (default: 900)
 Read extra arguments from standard input, one per line until EOF/Ctrl\-D
 (recommended for sensitive information such as passphrases)
 .SH COPYRIGHT
-Copyright (C) 2009-2016 The Bitcoin Core developers
+Copyright (C) 2009-2017 The Bitcoin Core developers
 
 Please contribute if you find Bitcoin Core useful. Visit
 <https://bitcoincore.org> for further information about the software.
 The source code is available from <https://github.com/bitcoin/bitcoin>.
 
 This is experimental software.
 Distributed under the MIT software license, see the accompanying file COPYING
-or <http://www.opensource.org/licenses/mit-license.php>.
+or <https://opensource.org/licenses/MIT>
 
 This product includes software developed by the OpenSSL Project for use in the
-OpenSSL Toolkit <https://www.openssl.org/> and cryptographic software written
-by Eric Young and UPnP software written by Thomas Bernard.
+OpenSSL Toolkit <https://www.openssl.org> and cryptographic software written by
```

Comment:
Why there are formatting changes likes this one ("by" moved to the previous line)?